db/models/topics.models.js

```diff
@@ -140,6 +140,23 @@ exports.changeVotesById = (article_id, votesToChange) => {
       } else {
         return PromResults[1].rows;
       }
-    
   });
 };
+
+exports.removeCommentAtId = (comment_id) => {
+  let queryString = `DELETE FROM comments `;
+  const queryValues = [];
+  const queryPromises = [];
+  queryString += ` WHERE comment_id = $1 RETURNING * `
+  queryValues.push(comment_id);
+  queryPromises.push(checkExists("comments", "comment_id", comment_id));
+  queryPromises.push(db.query(queryString, queryValues));
+  return Promise.all(queryPromises).then((PromResults) => {
+    if (queryPromises.length === 1) {
```

Comment:
The length of queryPromises will always be 2 since you're always pushing 2 promises to it, so this conditional check doesn't really make sense